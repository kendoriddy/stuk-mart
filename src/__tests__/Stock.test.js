import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Stock from '../components/Stock';
import store from '../redux/configureStore';

const CompanyProvider = () => (
  <Provider store={store}>
    <Stock />
  </Provider>
);

describe('renders the company stock data', () => {
  it('renders properly', () => {
    const component = renderer.create(<CompanyProvider />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
