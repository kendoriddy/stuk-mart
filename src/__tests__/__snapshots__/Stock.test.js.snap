// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the company stock data renders properly 1`] = `
<d1>
  Loading...
</d1>
`;
